packages/service/src/service.ts

```diff
@@ -87,8 +87,8 @@ export default class WdioImageComparisonService extends BaseClass {
     }
 
     beforeTest(test: Frameworks.Test) {
-        this.#currentFile = test.file
-        this.#currentFilePath = resolve(dirname(test.file), FOLDERS.DEFAULT.BASE)
+        this.#currentFile = test.filename
```

Comment:
This might work for Jasmine, but not for Mocha, I need to check this first before we can approve this